Integrations/integration-DemistoRESTAPI.yml

```diff
@@ -0,0 +1,98 @@
WYwAUYUAWIUAUoUAUIYAVYoAUIUAUIYAUIYAUYUAVYgAUIUAU4UAUIUAUIYAUoUAU4YAUYYAUIUAUIUAUYYAUIUAUYUAUIYAUIYAUIYAUIUAUIYA65oAVYUAUIcAUoUAUYUAUIYAUocAUIUAUIUAUIUAUoYAUYUAUYYAUIUAUYUAUIYAUYUAUYcAUIcAUoUAUYYAUIoAUIYAUIYAUIUAUYYAUIYAUYUAUIYAUYUAUYYAW5EAUIUAUYUAUIYAUYUAUIUAUYUAUIYAUYUAUYYAUIUAUIUAUIUAUYYAUIUAU4gAUIYAUIUAUoUAUYYAUIUAUYYAUIYAUIYAUoUA7JsA65oAUIUAUIYA65oAUIUA65p83hqwAAAA5XRSTlMAwzyIxSSwyRVkoVhsAinw/hwITLgKSEcBycX6ugUd/Q3p9h4E0vyv1Pv5+OIMB+oW6HDnSgYL4QnH7TKFA+5J9zTO3s/Z4Dmofk0nXbfI5cvGplY+1xLVXrQgNTpg2lrfEbLs9BN/7+SkDxqRRMqAqYpDlRto8K5SjVzR40A4EFGxNtBPbXTzv8Qh2xS2FyhGGOuO03otmz3ymEE3ffWLmsGTvMJyfImbKjNz1nsirM2CVyxunqomuVV1UK0jq4y9h6ezktyBDt1FpZ3mhpxUkLU/ZaPxK3mfdmGZg5RfO725olnhw9wAgAAABq1JREFUeNrt3OdbFEcYAPAXA4cYT4+ml1zuOLij9967IL0jIIiiokixxpLYo8bErrEk1vTee++9996Tyd+RDyC3s3XmdnYvT5b3292+784wv8fbdXZmASlGfFLTmdAQEInpiDhmSdSVnVwERJEzOmSbrPLjH72C29Y0iVO4x35prU4Q7Z4tN//snGxhSTCiDME4uc+NNiXZxJMTl9xcdIP1ciZhC9/U5mgigRBqJ5Ko4ZZ4I5GdcUKhjyu/y2MtYW7IVaqILqWTQCgzSyOJ1PcIILKQSon5lQS9HBpjKzGST1JjctJJoIiHtJFAbyhDFHeplOhOJOpm/EyWEoXNZEVxLjoJhE5pI1HmUpTYiNRJLCLuaAY7CUc6aVWckyex2l++wLJXEwn0oRKEM1KdRF8CcUdjjrGSsFeQl8XyJPwdBQrXtAWaSFh6FSQ+QaokUnIpxjLSyUhiH01dKU8CQi3yBbdoIoHulodwRKmTuJFqMJ9gI5GcSlO31cqTgEvyBYlpmkigLbISq5AqiZQSqsFc7mQisZiusJcv4W6VLwjSRuKgXQai3KJOIotyNNuZSKylKyziS4BL/lLRo40E75YFjzuQOonbKUezjYlEOl2hn0AC6mQvFZEaSRQMSEK8hFRK7KEczX4WEnYLXWGlUAJqA7kRjc+ZWOxSI1oSKBcLlQSvk4KwV6mVwC+dnSK9i4vALhSe0jRBajU38x3B4Wxxw5hoYaOdmFWiiAQvFizFzhksNaImIA4xiYRHJZKfRiol3Ny/2PKCaCNfxnNP4JTp+0xu4pXSeQHcvKhysZRdGIWyBCRj/60K0EgCvSqeu3OZWolw7rEWiS7dxE2azlpCYmziKCUgVhcJ1Cea+xpiKnGNRJdMmkrMEM/xp5WYoY9EvljqgQR9JFaEcWLY4BKif9ezSB8J4jCExJIU4VMFNCXhAwn0oiCzdUrCJxKbDvES29GUhE8k0J28R8+VUxI+koivJ5vNnpKQk5gXJBrBVKtzdnPTBlcykQjhHntYvJdB3WP1xf8XCYmYQ7dOajsn7W3ERAL2Ez6s2/CgsSVaLnI/PT65EAuew+Yeb/Neop90PtTy6SEjS5guYPnd4pMs6AfvJTaTz01/fNrIEnlJ3I/bLq/D68BOUxXivQT5EhuESnYYWAL+wgoeGc+xtmDfZpm9lxgso6DITDOwhBVbs7j/CAAAXI+dpRFUSMA6modnYQaWgPPYI5OfAADwnyzbUVUSA8toKOoMLAF/Yk8avwWAU/xlVmokYH08hUSgkSWCL+I/RZCG3bSmDquUgI00z/fLDSwBtVjNMTiLff4K1ErAruXkEheMLBG+jftV9OkYbB7FrF4CjvZQLD0ysARv7WIm9ulWYCAB0PHMCTKJTIPOAI5LWJskB6YG2EgAFD/wQYPEDOA87i2D2Ziz4hN1O6SuqVEuZhIyga3tGDS0hHBB+ESsAd0ljhhbQrBJYjySwvWXOG5sCVgjKvEk6C8xYnCJ8CG5O0o9Ja41uARv0g8hhFBEoS8kHD6RmMuJ876VsAYKJM6AeomUxsNf9Ly1dLcpdvPLj91LIuHyiQRBxOokIVzxtymNgQT9CuWF/1EJjVftc+vaeBL3AGsJqYlWbNX+AdYSJiYSvJ0smkrw7mSr7CwkcrALz1zRHq2P0fSKLb6ThSfx+1WeuB8ApvH2IPF2d0nvKfJ2dxcm+C7+yBRYSGB7ipClS6R3XdjuLoudtYRgd1etQKIMZv/jiasBIFRhIYqdcgZwImaRSeRw3zlyGJhIQCSii2UsZgDl90pa6vgSlbQS6aCpBDRwHpnWM5KooJQIYyHhjpBto8DFk/CjlVirsYR19eQBfDeqConPKSUyWEhApnwjrW5coohWYrHGEtA3uThymJXEdjqIiFeYSLQpNHMJl+illEhN1loC3pz4PghYSRT3U0nUMFlRAFsUmrGEciW2Wikl9oHmEq7xO9kKMzMJ+Izqn8RTbCRS3ldoqMDBkSgFOokKu/YS8LPnkSkjCXieQmIFsJGAOqWW/D3XxFigk0h3gA4SOc1iPxGqJAYOEnf0dTMrCdhA3Gick06iuRD0kIBfEYpyMJWAgCrCft6VDMwkzKtIIVxAJZE/AvpIWFtEfiJUzorvbCTq5kfZwE4CrBlEb+SceEspqURugxl0koDfcvNYSwDs7VTs5J4O5TkzGgmA+u8V3wU5+ebev2d74g9xCVtS0+g5t9JMHkMJCAD2EmCef59/tdTA2Kr91hWSzF7SSQAc//Hrk+kSy3Kxt1n/CyHOd2n4DBfNAAAAAElFTkSuQmCC
```

Comment:
what is the image you used? like the one for the Incidents generator right?

Reply:
@gshriki the logos should be the same in all integrations:

![image](https://user-images.githubusercontent.com/5429033/33010593-b7ecb9f2-cde3-11e7-96d6-24ada5eadb1a.png)

![image](https://user-images.githubusercontent.com/5429033/33010604-be8144f4-cde3-11e7-9033-40401bc05866.png)